test/Common/DurableTaskEndToEndTests.cs

```diff
@@ -2146,6 +2147,48 @@ public async Task Purge_Partially_History_By_TimePeriod(bool extendedSessions)
             }
         }
 
+
+        /// <summary>
+        /// End-to-end test which validates that incorrect task hub name throws instance of <see cref="ArgumentException"/>.
+        /// </summary>
+        [Theory]

```

Comment:
Instead of `[Theory]`, let's use `[Fact]` and remove the `[InlineData]` attributes.